src/core/Proc/Async.pm6

```diff
@@ -118,6 +117,13 @@ my class Proc::Async {
     has $!encoder;
     has @!close-after-exit;
 
+#?if moar
+    has atomicint $.started = 0;
+#?endif
+#?if !moar
+    has Bool $.started = False;
+#?endif
```

Comment:
We should have it as the `Bool` on all platforms. The `started` attribute is exposed API (due to the accessor); we should not be changing it to start returning an integer.

Reply:
Changed it back with most recent commit